modules/database/src/adapters/DatabaseAdapter.ts

```diff
@@ -8,6 +8,17 @@ export abstract class DatabaseAdapter<T extends SchemaAdapter<any>> {
   registeredSchemas: Map<string, ConduitSchema>;
   models?: { [name: string]: T };
 
+  /**
+   * Checks if the database adapter includes the declaredSchema model
```

Comment:
Minor nitpicking, but explain what the function does rather than how it does it.
It's a lie waiting to happen.